client/controller/client/http.go

```diff
@@ -111,7 +111,7 @@ func checkForErrors(res *http.Response, body string) error {
 	bodyMap := make(map[string]interface{})
 
 	if err := json.Unmarshal([]byte(body), &bodyMap); err != nil {
-		return err
+		return errors.New("\n" + body + "\n" + res.Status + "\n")
```

Comment:
> is there a reason for this ordering?

I'm following the convention of the JSON-formatted messages, which have the fields, then the status code. See elsewhere in the same test. I thought it was preferable to be internally consistent, although I would have put the status code first in both cases...